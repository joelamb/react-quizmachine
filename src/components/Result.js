import React from 'react';

const Result = ({ correctAnswer, questionsAnswered }) => (

  < div key={questionsAnswered} >
    {
      correctAnswer &&
      <React.Fragment>
        <p>Yes</p>
        <audio src="/static/assets/yeah_baby.mp3" autoPlay="true"></audio>
      </React.Fragment>
    }
    {
      correctAnswer !== '' && !correctAnswer &&
      <React.Fragment>
        <p>No</p>
        <audio src="/static/assets/dang.mp3" autoPlay="true"></audio>
      </React.Fragment>
    }
  </div >
)

export default Result;