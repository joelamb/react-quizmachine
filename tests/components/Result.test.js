import React from 'react';
import Result from '../../src/components/Result';
import renderer from 'react-test-renderer';

describe('Result', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(<Result />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
})