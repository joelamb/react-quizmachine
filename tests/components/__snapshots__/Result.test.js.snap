// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result renders correctly 1`] = `
<div
  className="result"
>
  <audio
    autoPlay="true"
    onEnded={[Function]}
    onLoadStart={[Function]}
    src="/static/assets/dang.mp3"
  />
  <div
    className="result__incorrect"
  >
    <img
      alt=""
      src="/static/assets/dang.jpg"
    />
  </div>
</div>
`;
